.gitignore

```diff
@@ -9,8 +9,6 @@
 # Artifacts
 build
 node_modules
-package-lock.json
-package.json.decrypt
```

Comment:
What does the `package.json.decrypt` file do? That sounds like something that might be good to keep out of version control.

Reply:
In trying to work out what was going wrong,
we synced the .gitignore files from the ESDK JS.

This is from that file.

Reply:
<from offline discussion/>

Specifically, it sounds like this was something that we added because of an artifact that a manual test created, not something that we expect to ever exist again.

I wanted to make sure that we were not blindly removing something that was protecting us from an unintended leak from a build tool or similar.